src/blocks/_pro/post-category-badge/variations.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+
+const variations = [
+	{
+		name: 'category-badge',
+		title: __('Category Badge', 'vk-blocks-pro'),
+		isDefault: true,
+		attributes: {
+			textAlign: 'center',
+			style: {
+				typography: {
+					fontSize: '0.8rem',
+					fontWeight: '700',
```

Comment:
```suggestion
					fontWeight: '500',
```
@mthaichi ここ普通の太さの方がよくないです？ VK Blocks Pro の投稿リストも太くしてないので